### PR TITLE
Use structured Bank Information

### PR DESCRIPTION
**Pull request by kewitz**
Requires opencollective/opencollective-api/pull/3790
Resolves opencollective/opencollective/issues/3093

- [x] Store bank information as PayoutMethod
- [x] Render bank information on the template

### CODE REVIEW

File: components/edit-collective/UpdateBankDetailsForm.js [hunk 1]
```diff
@@ -65,24 +62,25 @@ Please note that it will take a few days to process your payment.`,
...
                 label={intl.formatMessage(this.messages['bankaccount.instructions.label'])}
                 htmlFor="instructions"
-                width={500}
+                width="100%"
                 height={300}
                 onChange={e => this.onChange('instructions', e.target.value)}
-                defaultValue={
-                  get(value, 'instructions') || intl.formatMessage(this.messages['bankaccount.instructions.default'])
-                }
+                defaultValue={value || intl.formatMessage(this.messages['bankaccount.instructions.default'])}
```
Comment (Betree) [inline components/edit-collective/UpdateBankDetailsForm.js:73]: Doesn't this code generate a warning saying that the variables (`{amount}`, `{OrderId}`) are not set?

Comment (kewitz) [reply components/edit-collective/UpdateBankDetailsForm.js:73]: Nope.

Comment (kewitz) [reply components/edit-collective/UpdateBankDetailsForm.js:73]: No, it doesn't. It just renders it as plain text.

**Comment by kewitz** [comment]
Rebased.

**Comment by kewitz** [comment]
Adding some previews here to make it easier to check.
@sbinlondon I tested the normal flow and everything worked normally, I suspect this error is a special case using the test admin I have locally. Are you aware of this error?

Previews:
localhost_3000_brusselstogetherasbl_accept-financial-contributions_myself_bank

localhost_3000_brusselstogetherasbl_accept-financial-contributions_myself_bank(iPhone X)

**Pull request merged**